tests/simpleKVBC/TesterReplica/internalCommandsHandler.cpp

```diff
@@ -88,17 +96,29 @@ void InternalCommandsHandler::execute(InternalCommandsHandler::ExecutionRequests
                                    req.outActualReplySize,
                                    req.outReplicaSpecificInfoSize);
     } else {
+      bool isBlockAccumulationEnabled = (ReplicaConfig::instance().blockAccumulation &&
+                                         (!pre_execute && (req.flags & bftEngine::MsgFlag::HAS_PRE_PROCESSED_FLAG)));
+
       res = executeWriteCommand(req.requestSize,
                                 req.request,
                                 req.executionSequenceNum,
                                 req.flags,
                                 req.maxReplySize,
                                 req.outReply,
-                                req.outActualReplySize);
+                                req.outActualReplySize,
+                                isBlockAccumulationEnabled,
+                                verUpdates,
+                                merkleUpdates);
     }
+
     if (!res) LOG_ERROR(m_logger, "Command execution failed!");
     req.outExecutionStatus = res ? 0 : -1;
   }
+
+  if (!pre_execute && (merkleUpdates.size() > 0 && verUpdates.size() > 0)) {
```

Comment:
It may be that only merkleUpdates or verUpdates are not empty, in this case data should be written. I think that AND should be changed to OR

Reply:
yes. I have updated code with OR check

Reply:
Thanks! Please next time you need to force push, use --force-with-lease, it will try to preserve previous commits IDs and to make further reviewing more convinient.

Reply:
Thanks.Actually I did use--force-with-lease but not sure why the older commits are not visible. I will take a look next time as well.